frontend/src/streamDetails/__tests__/StreamLogsCard.test.jsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import Enzyme from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import { describe, expect, it } from "@jest/globals";
+
+import { Grid } from "@material-ui/core";
+import DashboardCard from "../../general/dashboard/DashboardCard";
+import StreamLogsCard from "../StreamLogsCard";
+import LogsTableWrapper from "../../loglist/LogsTableWrapper";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("<StreamLogsCard /> functional component", () => {
+  let wrapper;
+
+  const dummyLogSource = {
+    getStreamLogs() {
```

Comment:
can be simplified to `getStreamLogs: jest.fn()`

Reply:
not gonna bother with this one since logs are part of sharon's task so i'm expecting a lot of this code to disappear lol

Reply:
lmao valid